shell/common/surface_frame.cc

```diff
@@ -0,0 +1,62 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/common/surface_frame.h"
+
+#include "flutter/fml/logging.h"
+
+namespace flutter {
+
+SurfaceFrame::SurfaceFrame(sk_sp<SkSurface> surface,
+                           bool supports_readback,
+                           const SubmitCallback& submit_callback)
+    : submitted_(false),
```

Comment:
Initialize these in the header please. That way, if/when new ctors are added, they don't all have to remember to initialize all ivars.

Reply:
I see you just moved this TU. I bet this is an old TU. I'd recommend updating it nonetheless.

Reply:
Done.